app/models/ticket_stakeholder.rb

```diff
@@ -34,6 +30,14 @@ def user_stakeholder?
     stakeholder_type == "User"
   end
 
+  def actions_allowed
+    ticket.metadata.actions_allowed(self)
+  end
+
+  DEFAULT_SERIALIZE_OPTIONS = {
+    methods: %w[stakeholder actions_allowed],
```

Comment:
Not really related to the intention of your current PR, but if you're touching `DEFAULT_SERIALIZE_OPTIONS` anways, then you can also move `stakeholder` to `include` instead of `methods`.

Reply:
Okay changed. Thank you.